crates/matrix-sdk/src/sliding_sync/list/mod.rs

```diff
@@ -94,6 +94,12 @@ impl SlidingSyncList {
         self.inner.state.read().unwrap().clone()
     }
 
+    /// Check whether this list requires a [`http::Request::timeout`] value.
```

Comment:
Maybe expand on that comment too, or link to the other comment where we explain why we set the timeout value or not?

Reply:
Yup, indeed!